src/XenCert/XenCert

```diff
@@ -187,6 +179,27 @@ def main():
         sys.exit(0)
     else:
         sys.exit(1)
-    
+
+
+def GetStorageHandler(g_storage_conf):
```

Comment:
if possible I suggest move it to module StorageHandler

Reply:
Done